project/ftlabs-ftda.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Page Uploader</title>
+        <link rel="stylesheet" href="https://www.ft.com/__origami/service/build/v2/bundles/css?modules=o-fonts,o-icons" />
+        <style>
+	        body {
+				background: #FFF1E5;
+				font-family: MetricWeb, sans-serif;
+			}
+
+			.o-icons-icon--tick {
+				width: 30px;
+				height: 30px;
+				display: inline-block;
+				vertical-align: middle;
+				margin-right: 10px;
+			}
+        </style>
+    </head>
+    <body>
+    	<h1>FT Digital Archive Authenticator</h1>
+    	<p><i class="o-icons-icon o-icons-icon--tick"></i>Success</p>
+    	<p>You have been authenticated. Go back to the application and click 'Continue' to activate your credentials.</p>
+    </body>
+    <script async type="javascript" src="https://www.ft.com/__origami/service/build/v2/bundles/js?modules=o-fonts,o-icons"></script>
```

Comment:
Does this <script> need to be `async` if it's the last thing loaded in the document. Do we get a benefit from it? (asking for a friend)

Reply:
From Origami documentation.